snippets/get_transaction.py

```diff
@@ -1,49 +1,47 @@
 """A snippet that walks us through getting a transaction."""
-from xrpl.asyncio.clients import AsyncWebsocketClient
+from xrpl.clients import JsonRpcClient
 from xrpl.models.requests import Ledger, Tx
 
 
-async def async_get_transaction() -> None:
+def get_transaction(client: JsonRpcClient) -> None:
```

Comment:
Why did you move the client to outside the function?

Reply:
Actually, I don't know if these have to be functions at all - I think these could just be scripts.

Reply:
If I don't make it a function, I think it tries running it when I make docs

Reply:
Also, I can move client into function. No real reason I put it outside.

Reply:
Actually, I think I can make it not run on make docs